cognite/neat/_rules/models/_base_rules.py

```diff
@@ -233,6 +233,20 @@ def as_data_model_id(self) -> dm.DataModelId:
     def as_identifier(self) -> str:
         return repr(self.as_data_model_id())
 
+    @classmethod
+    def default(cls) -> "BaseMetadata":
+        """Returns a default instance of the metadata model."""
+        return cls(
+            space="pleaseUpdateMe",
+            external_id="PleaseUpdateMe",
+            version="v1",
+            name="Please Update Me",
+            description="Please Update Me",
+            creator=["NEAT"],
+            created=datetime.now(),
+            updated=datetime.now(),
```

Comment:
Ensuring that these are the same.
```suggestion
            created=now,
            updated=now,
```